src/global/core-styles.scss

```diff
@@ -27,6 +27,15 @@
     --lime-elevated-surface-background-color: rgb(var(--contrast-100));
 }
 
-@include mixins.in(dark-mode) {
-    --lime-elevated-surface-background-color: rgb(var(--contrast-200));
-}
+// ⚠️ This section below is commented out.
+// It's contents are moved to src/style/color-palette-extended.css
+//
+// Because we don't want to give dark-mode-specific shadows to those
+// clients that only import src/style/color-palette-extended-light-mode-only.css
+// Because if their OS is in dark mode, not their app,
+// The shadows will get rendered too dark, due to `@media (prefers-color-scheme: dark)`
+// rule from the mixin. The day we generate `.css` files from these `.scss` files,
+// we can do it differently.
```

Comment:
What and how can we do things differently? These files are of course already being converted to .css, since browsers don't read .scss-files. But I suspect you refer to generating separate .css-files with _just_ the variable definitions?

Reply:
There are a few ways to treat this issue.

<details><summary>Dream scenario</summary>
I would have liked to have a file called `theme-light.css` that automatically is generated from a `theme-light.scss` file when the build goes through. Along with `theme-light-and-dark.scss` file, that also generates its own CSS version.

These files would then contain variables that we need, both colors, and shadows, and other probable things which we may need, like the elevated-surface variable.

But this could mean a breaking change, and consumers would now need to import other CSS files with other names. Maybe hard. I don't know.
</details>

<details><summary>More pragmatic way</summary>
We just make sure that there is an SCSS file for `color-palette-extended-light-mode-only.scss` and `color-palette-extended.scss` which imports colors and shadows and such variables in itself. These files would automatically then get converted to CSS when the build goes through.
</details>

<details><summary>Ultimate dream scenario</summary>
That we wouldn't have to deal with `force-dark` and `force-light` at all. We could just somehow tell the browser that user for this page prefers light or dark color them, no matter what their OS setting is saying.

We would anyway need to have some auto CSS generation, because that would help us keep different variables in different files and organize stuff better.
</details>

Reply:
In any case, we can let this go for now and open new issues for any other solution.